package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "lint": "yarn workspaces foreach -A run lint",
     "clean": "yarn workspaces foreach -A run clean",
-    "build": "yarn run clean && yarn workspaces foreach -A run build:all",
+    "build": "yarn run clean && yarn workspaces foreach -A -t run build:all",
```

Comment:
Just curious, did you face any problem running the build without the "--topological" flag?

Reply:
yes, this is why I added